source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -1235,6 +1235,55 @@ the pool's generation number then error handling MUST continue according to
 stale and the client MUST NOT update any topology state.
 (See `Why ignore errors based on CMAP's generation number?`_)
 
+Error handling behavior
+~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
This section still needs to be fleshed out a bit more. My goal is to replace the long-winded "Network error when reading or writing" and "not master" and "node is recovering" error handling sections with a more straightforward section focused on pseudocode.